src/mahoji/commands/sell.ts

```diff
@@ -112,6 +112,45 @@ export const sellCommand: OSBMahojiCommand = {
 			return `You exchanged ${moleBank} and received: ${loot}.`;
 		}
 
+		if (
+			bankToSell.has('Abyssal blue dye') ||
+			bankToSell.has('Abyssal green dye') ||
+			bankToSell.has('Abyssal red dye') ||
+			bankToSell.has('Abyssal lantern')
+		) {
+			const abbyBank = new Bank();
+			const loot = new Bank();
+			if (bankToSell.has('Abyssal lantern')) {
+				abbyBank.add('Abyssal lantern', bankToSell.amount('Abyssal lantern'));
```

Comment:
Do these show in the autocomplete? They should maybe be reverts or useable items.

Reply:
We don't have autocomplete for sell command